### PR TITLE
Rename variables to keep naming scheme

### LINKED ISSUE #159: Variable naming conventions
The newest version of `ansible-lint` says, variables within roles should use the role name as a prefix.

For the most part we already stick to that rule. But we have a bunch of variables that are global to the whole collection. They are mostly used for interaction between the roles in the collection. These are prefixed with `elastic_`.

How should we deal with this new rule?

* Just put an exception into the `ansible-lint` configuration?
* Rename the variables so that they are local to every role? If we need interaction we need a lot of `set_fact` for synchronization.
* Come up with a completely different solution?
* Find a way to have "global variables" in collections? A short search didn't reveal any option to do that.

### PR DESCRIPTION
**Pull request by widhalmt**
fixes #159

### CODE REVIEW

**Comment by widhalmt** [comment]
I made this a draft because there are still some variables missing in the renaming. We'll discuss how to proceed with them in #159

**Comment by widhalmt** [comment]
We opened a discussion around that issue upstream: https://github.com/ansible/ansible-lint/discussions/3451

To move fast, we now put this on the warn_list and come back to it, when
the discussion is resolved. Hopefully to remove the rule completly from
`ansible-lint.yml`.

**Review by dgoetz — CHANGES_REQUESTED** [review]
If the collection is accepted as prefix in the upstream discussion everything related to the CA shared by the complete stack should be prefixed `elasticstack_`, this goes for unchanged variables like `elastic_ca_pass` and some changed in the PR like `elastic_ca_will_expire_soon`/`elasticsearch_ca_will_expire_soon`. So I would keep thise unchanged until then.

**Comment by widhalmt** [comment]
@dgoetz I hope, I now caught all of the missing variables. Honestly, I feel a slight hint of panic when I think of merging this PR and all the others that are still open. 😱

**Comment by widhalmt** [comment]
Some of the checks are failing because of timeouts. As long as all `lint` checks and the required checks work, we should be safe.

File: docs/role-kibana.md [hunk 1]
```diff
@@ -16,23 +16,23 @@ Role Variables
...
 * *kibana_tls_key*: Path to the key Kibana should use when communicating with clients (default: `/etc/kibana/certs/key.pem`)
 
-* *kibana_security*: Activate TLS and authentication when connecting to Elasticsearch. **Note**: Only works when `elastic_stack_full_stack` is enabled. (default: `true`)
+* *kibana_security*: Activate TLS and authentication when connecting to Elasticsearch. **Note**: Only works when `elasticstack_stack_full_stack` is enabled. (default: `true`)
 
 These variables are identical over all our elastic related roles, hence the different naming scheme.
 
-* *elastic_stack_full_stack*: Use `ansible-role-elasticsearch` as well (default: `false`)
-* *elastic_elasticsearch_http_port*: Port of Elasticsearch http (Default: `9200`)
+* *elasticstack_stack_full_stack*: Use `ansible-role-elasticsearch` as well (default: `false`)
```
Comment (afeefghannam89) [inline docs/role-kibana.md:23]: How about remove the first stack word?
```suggestion
* *elasticstack_full_stack*: Use `ansible-role-elasticsearch` as well (default: `false`)
```

**Comment by afeefghannam89** [comment]
@widhalmt woo It is a big type work ✍️ Thanks I see the PR is ok. The only suggestion is the above, there are three stack words in the variable name. You can take it or remove it.

**Review by afeefghannam89 — APPROVED** [review]

**Review by dgoetz — APPROVED** [review]
Naming looks consistent now within the collection and with the guideline (including our proposed addition). It creates some clunky names but that seems unavoidable with the goal in mind. Nitpicking could be done in form of complaining some not being in alphabetical or logical order, but I am ok with this.

**Comment by widhalmt** [comment]
Some of them aren't in alphabetical order on purpose. I tried to group them logically. :-)

**Pull request merged**